Modules/Scripted/VectorToScalarVolume/VectorToScalarVolume.py

```diff
@@ -76,6 +103,26 @@ def setup(self):
     # Add vertical spacer
     self.layout.addStretch(1)
 
+  def onRGBCheckBox(self, state):
+    if (state == 2): # Checked
+      # Disable other box.
+      self.singleComponentBox.setChecked(False)
```

Comment:
I don't know exactly how the UI looks like, but if you have to do this, then probably radio button would be a better widget to use.

Reply:
Definitely right, thanks @cpinter

Reply:
I couldn't get the ratio buttons to be exclusive just with the parent (maybe because there are frames?). Anyway, I created an exclusive QButtonGroup, and the buttons there. It is a few lines more verbose but explicit. Let me know what you think.

Reply:
Common parent and setAutoExclusive(true) usually does the trick. Maybe you forgot the latter? Jc says it's not needed, but maybe it depends on the order you do things. What happens if you set it to true explicitly? That said I wouldn't mind the button group if that's how you can make it work.